PluginsAndFeatures/AddLibrary/AppInsights/com.microsoft.applicationinsights.feature/pom.xml

```diff
@@ -6,7 +6,7 @@
 	<relativePath>../../../azure-toolkit-for-eclipse/pom.xml</relativePath>
     <groupId>com.microsoft.azuretools.wap4ej.build</groupId>
     <artifactId>parent</artifactId>
-    <version>3.12.0-SNAPSHOT</version>
+    <version>${eclipse.version}</version>
```

Comment:
azure.toolkit.eclipse.version

Reply:
use "azuretool.eclipse.version", since the project has already use like "azuretool.sdk.version"